src/templates/design-system-index.js

```diff
@@ -45,7 +49,7 @@ const Template = ({children,Pagedata,pageContext,location, lastModified}) => {
               { Pagedata.components.hero && <Hero {...Pagedata.components.hero} pageContext={pageContext} {...Pagedata.seo}></Hero>}
               {Pagedata.components.sectionsEditorial && Pagedata.components.sectionsEditorial.map((section,index) => {
                 return(
-                  <SectionEditorial key={"sectionEditorial-"+index} {...section}/>
+                  <SectionEditorial key={"sectionEditorial-"+index} {...section} componentViewerData={Object.assign({}, { variants: variantMock })}/>
```

Comment:
```suggestion
                  <SectionEditorial key={"sectionEditorial-"+index} {...section} componentViewerData={{ variants: variantMock }}/>
```
Unless I'm missing something. Does it need to be a copy?